site/src/component/PrereqTree/PrereqTree.tsx

```diff
@@ -72,7 +72,7 @@ const PrereqTreeNode: FC<TreeProps> = (props) => {
           }
           content={
             props.prerequisiteNames[
-              prereq.prereqType === 'course' ? prereq.courseId.replace(/ /g, '') : (prereq.examName ?? '')
+              prereq.prereqType === 'course' ? prereq.courseId.replace(/ /g, '') : prereq.examName ?? ''
```

Comment:
while this is correct, the precedence of `? :` over `??` is not obvious to everyone, so I think this is a change that should be reverted

(unless that was a prettier auto-format)